src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -634,6 +634,29 @@ public async Task Property_completions_required_first_and_selected()
                 '|');
         }
 
+        [TestMethod]
```

Comment:
>TestMethod [](http://example.com/codeflow?start=9&length=10)

Should we also update some of the baseline tests to add the new operator?

Reply:
I added a usage to the `TypeDeclarations_LF` baseline (mostly because an array of a union type is the easiest way to trigger a nullable type prior to #9533 or #9454 landing), and it looks like one of the invalid expressions from `InvalidExpressions_LF` is now valid (at least at a syntactic level).